Index.html

```diff
@@ -115,13 +123,16 @@ <h2>Skills</h2>
             </section>
         </div>
         <section class="grid-item recent-work">
-            <h4>MY RECENT WORK</h4>
+            <h4>My recent work</h4>
```

Comment:
Figma Design                |  Your design     
:-------------------------:|:-------------------------:
![bg1](https://user-images.githubusercontent.com/69638013/155659099-3fbe8634-04de-4002-a0ed-cea6e599d7ab.PNG)  | ![image](https://user-images.githubusercontent.com/69638013/155659169-79ee7a7a-d983-4879-a76b-1dcca7bce5d1.png)


- [ ] Please align my recent work header and   border-bottom start to start where the first image start check the Figma design, check screen shot above